libs/barista-components/filter-field/src/filter-field.ts

```diff
@@ -1119,6 +1119,10 @@ export class DtFilterField<T = any>
               recentRangeValue.operator as DtFilterFieldRangeOperator,
             );
           }
+          if (isDtFreeTextDef(this._currentDef)) {
+            this._inputValue = recentRangeValue.toString();
```

Comment:
Not sure what the `recentRangeValue` has to do with a `free-text` - can you clarify it for me?

Reply:
Renamed to initialRecentlyRemoved

Reply:
This does not solve the issue that we want to prefill free-text not ranges! You are reading the range value from the removed ones an setting it into the free-text input. This does not make sense

Reply:
The check at L1122 tells me that the current definition is a free-text, so I'm rest assured that the removed item is the string to prefill the input

Reply:
got it. thank you